lib/runtime-core/Cargo.toml

```diff
@@ -8,46 +8,46 @@ repository = "https://github.com/wasmerio/wasmer"
 edition = "2018"
 
 [dependencies]
-nix = "0.14.0"
+nix = "0.14.1"
 page_size = "0.4.1"
 wasmparser = "0.35.1"
-parking_lot = "0.7.1"
-lazy_static = "1.2.0"
-indexmap = "1.0.2"
+parking_lot = "0.9.0"
+lazy_static = "1.3.0"
 errno = "0.2.4"
-libc = "0.2.49"
+libc = "0.2.60"
 hex = "0.3.2"
-smallvec = "0.6.9"
+smallvec = "0.6.10"
 bincode = "1.1"
 colored = "1.8"
 
+[dependencies.indexmap]
+version = "1.0.2"
+features = ["serde-1"]
+
 # Dependencies for caching.
 [dependencies.serde]
-version = "1.0"
+version = "1.0.98"
 # This feature is required for serde to support serializing/deserializing reference counted pointers (e.g. Rc and Arc).
 features = ["rc"]
 [dependencies.serde_derive]
-version = "1.0"
+version = "1.0.98"
 [dependencies.serde_bytes]
-version = "0.10"
+version = "0.11.1"
 [dependencies.serde-bench]
 version = "0.0.7"
 [dependencies.blake2b_simd]
-version = "0.4.1"
+version = "0.5.5"
 [dependencies.digest]
-version = "0.8.0"
-[dependencies.hashbrown]
-version = "0.1"
-features = ["serde"]
```

Comment:
This hashbrown `serde` feature was why I think we should re-test caching after removing `hashbrown`.